tidb-cloud/serverless-high-availability.md

```diff
@@ -0,0 +1,118 @@
+---
+title: High Availability in TiDB Cloud Serverless
+summary: Learn about the high availability architecture of TiDB Cloud Serverless. Discover Zonal and Regional High Availability options, automated backups, failover processes, and how TiDB ensures data durability and business continuity.
+---
+
+# High Availability in TiDB Cloud Serverless
+
+TiDB Cloud Serverless is designed with robust mechanisms to maintain high availability and data durability by default, preventing single points of failure and ensuring continuous service even in the face of disruptions. As a fully managed service based on the battle-tested TiDB Open Source product, it inherits TiDB's core high availability (HA) features and augments them with additional cloud-native capabilities.
+
+## Overview
+
+TiDB ensures high availability and data durability using the Raft consensus algorithm. This algorithm consistently replicates data changes across multiple nodes, allowing TiDB to handle read and write requests even in the event of node failures or network partitions. This approach provides both high data durability and fault tolerance.
+
+TiDB Cloud Serverless extends these capabilities with two types of high availability to meet different operational requirements:
+
+- **Zonal high availability (default)**: This option places all nodes within a single availability zone, reducing network latency. It ensures high availability without requiring application-level redundancy across zones, making it suitable for applications that prioritize low latency within a single zone. Zonal high availability is available in all regions that support TiDB Cloud Serverless. For more information, see [Zonal high availability architecture](#zonal-high-availability-architecture).
+
+- **Regional high availability (beta)**: This option distributes nodes across multiple availability zones, offering maximum infrastructure isolation and redundancy. It provides the highest level of availability but requires application-level redundancy across zones. It is recommended to choose this option if you need maximum availability protection against infrastructure failures within a zone. Note that it increases latency and might incur cross-zone data transfer fees. This feature is available in selected regions with multi-availability zone support and can only be enabled during cluster creation. For more information, see [Regional high availability architecture](#regional-high-availability-architecture).
+
+> **Note:**
+>
+> Regional high availability is in beta and only available in the AWS Tokyo (`ap-northeast-1`) region.
```

Comment:
```suggestion
```
This note is duplicated with https://github.com/pingcap/docs/blob/6cbeec597da53b37635f57acef69a4887841d5d1/tidb-cloud/serverless-high-availability.md?plain=1#L52-L59